src/mixins/legal-api-mixin.ts

```diff
@@ -255,7 +255,14 @@ export default class LegalApiMixin extends Mixins(FilingTemplateMixin) {
     return axios.get(url)
       .then(response => {
         if (response?.data) {
-          return response.data
+          const shareStructure = response.data
+          // Apply a type to share classes and series
+          shareStructure.shareClasses.forEach(shareClass => {
+            shareClass.type = 'Class'
+            shareClass.series.forEach(shareSeries => { shareSeries.type = 'Series' })
```

Comment:
If this "fix" is needed in the business snapshot, is the same code in filing-template-mixin.ts::parseBusinessSnapshot() now obsolete (around line 565) ?

Reply:
Yes, we only need to do this once this way.

Reply:
Fixed in next commit